src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -3947,6 +3951,15 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         SearchMissingCommand.raise(*this, args);
     }
 
+    til::CoordType TermControl::_searchScrollOffset() const
+    {
+        const auto displayInfo = DisplayInformation::GetForCurrentView();
+        const auto scaleFactor = _core.FontSize().Height / displayInfo.RawPixelsPerViewPixel();
+        const auto searchBoxRows = _searchBox->ActualHeight() / scaleFactor;

```

Comment:
📝: is this ever null? Like, if we `f3`/`findNext` immediately before actually opening the search box?

Reply:
conclusion: yes it's null-initialized, but everywhere that uses this first checks that it's non-null, so it's fine

Reply:
updated,  I ended up having to add the null check with the updates for only calculating when the font size or dpi changed,  the calculate method was now being called in initialize when the _searchBox was null.